### PR TITLE
Design/post new page

### PR DESCRIPTION
**Pull request by vasanthakumar-a**
Post new Page is designed as like as reddit.com/submit

### CODE REVIEW

File: app/assets/stylesheets/posts.scss [hunk 1]
```diff
@@ -59,3 +59,75 @@
...
+  text-align: center;
+  text-decoration: none;
+}
+.draft-btn:hover {
+  background-color: rgba(99,108,114,0.2);
+  cursor: pointer;
+}
+
+.post-tab-nav-link {
+	background-color: #fff;
```
Comment (iMagesh) [inline app/assets/stylesheets/posts.scss:93]: check the indentation here

File: app/assets/stylesheets/warning.scss [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  user-select: none;
+}
+.warning-nav-link.active {
+	background: #f6f7f8;
+	color: #007BFF;
+}
+.warning-nav-link.active a {
+	color: #007BFF;
+}
+.warning-nav-link a {
```
Comment (iMagesh) [inline app/assets/stylesheets/warning.scss:17]: indentation seems to be off here

File: app/javascript/includes/navbar.js [hunk 3]
```diff
@@ -29,6 +29,13 @@ $( document ).on('turbolinks:load', function() {
             $(this).addClass('active')
         }
     }
+    $('.warning-nav-link').click(function(event){
+        $(this).children(".fa-solid").toggleClass('fa-plus fa-check ');
+        $(this).toggleClass(' active');
+        $(this).children("#check").prop('checked',!$(this).children("#check").prop('checked'));
```
Comment (iMagesh) [inline app/javascript/includes/navbar.js:35]: check the indentation here as well

File: app/assets/stylesheets/warning.scss (hunk 2 above)
Comment (vasanthakumar-a) [reply app/assets/stylesheets/warning.scss]: indented scss.

**Comment by vasanthakumar-a** [comment]
indentation completed for all the scss file .

**Pull request merged**